src/main/webapp/components/Documents.js

```diff
@@ -0,0 +1,52 @@
+import { Component } from 'react';
+import { input, hh, h, div, p, hr, small, button } from 'react-hyperscript-helpers';
+import { Table } from './Table';
+import { Panel } from './Panel';
+
+ const headers =
+  [
```

Comment:
Nit: the spacing is off in this file. If you reformat the lines, you'll see a lot of whitespace changes.